lib/Mount/CollectiveFolderManager.php

```diff
@@ -149,13 +150,13 @@ private function getJailPath(int $folderId): string {
 	/**
 	 * @throws NotFoundException
 	 */
-	private function getRootFolderStorageId(): int {
+	private function getFolderRootId(): int {
 		if ($this->rootFolderStorageId === null) {
 			$qb = $this->connection->getQueryBuilder();
 
-			$qb->select('fileid')
-				->from('filecache')
-				->where($qb->expr()->eq('storage', $qb->createNamedParameter($this->getRootFolder()->getStorage()->getCache()->getNumericStorageId())))
+			$qb->select('f.fileid')
+				->from('filecache', 'f')
+				->where($qb->expr()->eq('storage', $qb->createNamedParameter($this->getRootFolderStorageId())))
 				->andWhere($qb->expr()->eq('path_hash', $qb->createNamedParameter(md5($this->getRootPath()))));
 
 			$this->rootFolderStorageId = (int)$qb->execute()->fetchColumn();
```

Comment:
Please rename this instance variable to `$this->folderRootId` to avoid confusion.

Reply:
Used `getRootFolderId` / `$this->rootFolderId` now as it is in line with the new `getRootFolderStorageId`